.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,42 @@
+name: Bug report
+description: Create a report to help us fix issues
+labels: ["Bug"]
+body:
+  - type: input
+    id: modbuild
+    attributes:
+      label: Mod Build
+      description: Found on top of the news screen.
+      placeholder: ex. 153
+    validations:
+      required: true
+  - type: input
+    id: gamebuild
+    attributes:
+      label: Game Build
+      description: Found under the title in the main menu.
+      placeholder: ex. 145.1
+    validations:
+      required: true
+  - type: textarea
+    id: desc
+    attributes:
+      label: Issue Description
+      description: Explain the issue in details.
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: Make a list of all the steps.
+      placeholder: 1. ...
+    validations:
+      required: false
```

```suggestion
      required: true
```
Make steps to reproduce required. And if there's none well, None they can put or something like that.

---

Solved